extras/free-books.md

```diff
@@ -87,6 +87,9 @@ Name | Author(s)
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author(s) 
+:-- | :--: 
+[Linear Algebra](https://www.amazon.com/Linear-Algebra-2nd-Kenneth-Hoffman/dp/0135367972) | Kenneth M. Hoffman
```

Comment:
It is right up on the google [here](http://uap.unnes.ac.id/ebook/mathematics%20ebook/Algebra/Linear%20Algebra,%202nd%20ed%20-%20Kenneth%20Hoffman,%20Ray%20Kunze,%201971.pdf)